app/models/datastreams/dams_resource_datastream.rb

```diff
@@ -599,13 +599,13 @@ def insertRelatedResourceFields ( solr_doc, prefix, relatedResource )
       related_json[:id] = resource.pid
 
       related_json = {:type=>resource.type.first.to_s, :uri=>resource.uri.first.to_s, :description=>resource.description.first.to_s}
-      Solrizer.insert_field(solr_doc, "related_resource_json", related_json.to_json)
+      Solrizer.insert_field(solr_doc, "#{prefix}related_resource_json", related_json.to_json)
       Solrizer.insert_field(solr_doc, "all_fields", resource.uri.first.to_s)
       Solrizer.insert_field(solr_doc, "all_fields", resource.type.first.to_s)
       Solrizer.insert_field(solr_doc, "all_fields", resource.description.first.to_s)
       if resource.type.first.to_s == "thumbnail"
         Solrizer.insert_field(solr_doc, "thumbnail", resource.uri.first.to_s)
-	  end
+	    end
```

Comment:
Actually the indent looks good on my sublime editor. Not sure why it shows different way on webpage. I just make sure empty space trimmed around the "end" keyword.